packages/p2p/src/pages/my-profile/block-user/block-user-empty/block-user-empty.tsx

```diff
@@ -27,9 +30,10 @@ const BlockUserEmpty = () => {
                 <MobileFullPageModal
                     body_className='block-user-empty'
                     height_offset='80px'
-                    is_modal_open={my_profile_store.active_tab === my_profile_tabs.MY_COUNTERPARTIES}
+                    is_modal_open={is_modal_open}
                     page_header_text={localize('My counterparties')}
-                    pageHeaderReturnFn={() => my_profile_store.setActiveTab(my_profile_tabs.MY_STATS)}
+                    pageHeaderReturnFn={setActiveTab(my_profile_tabs.MY_STATS)}
```

Comment:
would this actually invoke the function? Could we do something liek:
```suggestion
                    pageHeaderReturnFn={() => my_profile_store.setActiveTab(my_profile_tabs.MY_STATS)}
```

Reply:
Yeah, but we need to cover that in test, the coverage was below 70% because of this inline function